zerver/openapi/zulip.yaml

```diff
@@ -1232,6 +1266,10 @@ paths:
                       type: integer
                       description: The user's ID.
                       example: 1
+                    profile_data:
+                      type: object
+                      description: A object mapping the id of each custom profile field id to the
+                        value and rendered_value for it (if it exists) for the requesting user.
```

Comment:
@timabbott I definitely think that this could be worded better. But I also think that you already have a nice way in mind for this :slightly_smiling_face: